projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -1376,9 +1376,9 @@ export class IgxDatePickerComponent implements IDatePicker, ControlValueAccessor
 
         componentInstance.mode = this.mode;
         componentInstance.vertical = isVertical;
-        componentInstance.cancelButtonLabel = this.cancelButtonLabel;
+        componentInstance.closeButtonLabel = this.cancelButtonLabel;
         componentInstance.todayButtonLabel = this.todayButtonLabel;
-        componentInstance.datePickerActions = this.datePickerActionsDirective;
+        componentInstance.pickerActions = this.datePickerActionsDirective;
 
         // componentInstance.onClose.pipe(takeUntil(this._destroy$)).subscribe(() => this.closeCalendar());
         // componentInstance.onTodaySelection.pipe(takeUntil(this._destroy$)).subscribe(() => this.triggerTodaySelection());
```

Comment:
not sure why these two lines are commented out, but `cancel` and `today` buttons do nothing now in date picker